x-pack/legacy/plugins/siem/public/components/events_viewer/index.tsx

```diff
@@ -5,26 +5,35 @@
  */
 
 import { isEqual } from 'lodash/fp';
-import React, { useEffect, useState, useCallback } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { connect } from 'react-redux';
 import { ActionCreator } from 'typescript-fsa';
-import { WithSource } from '../../containers/source';
+import chrome from 'ui/chrome';
 import { inputsModel, inputsSelectors, State, timelineSelectors } from '../../store';
-import { timelineActions, inputsActions } from '../../store/actions';
-import { KqlMode, TimelineModel } from '../../store/timeline/model';
+import { inputsActions, timelineActions } from '../../store/actions';
+import { KqlMode, SubsetTimelineModel, TimelineModel } from '../../store/timeline/model';
 import { ColumnHeader } from '../timeline/body/column_headers/column_header';
 import { DataProvider } from '../timeline/data_providers/data_provider';
 import { Sort } from '../timeline/body/sort';
 import { OnChangeItemsPerPage } from '../timeline/events';
-import { Query, esFilters } from '../../../../../../../src/plugins/data/public';
+import { esFilters, Query } from '../../../../../../../src/plugins/data/public';
 
 import { EventsViewer } from './events_viewer';
 import { InputsModelId } from '../../store/inputs/constants';
+import { useFetchIndexPatterns } from '../../containers/detection_engine/rules/fetch_index_patterns';
+import { TimelineTypeContextProps } from '../timeline/timeline_context';
+import { DEFAULT_INDEX_KEY } from '../../../common/constants';
 
 export interface OwnProps {
+  defaultIndices?: string[];
+  defaultFilters?: esFilters.Filter[];
```

Comment:
@angorayc -- I've added this `defaultFilters` prop you can use to specify the necessary filters for the `Alerts Table`.

Reply:
nice add